drivers/platform/stm32/stm32_pwm.c

```diff
@@ -144,6 +144,36 @@ static int32_t stm32_init_timer(struct stm32_pwm_desc *desc,
 	case 6:
 		base = TIM6;
 		break;
+#endif
+#if defined(TIM7)
+	case 7:
+		base = TIM7;
+		break;
+#endif
+#if defined(TIM8)
+	case 8:
+		base = TIM8;
+		break;
+#endif
+#if defined(TIM9)
+	case 9:
+		base = TIM9;
+		break;
+#endif
+#if defined(TIM10)
+	case 10:
+		base = TIM10;
+		break;
+#endif
+#if defined(TIM11)
+	case 11:
+		base = TIM11;
+		break;
+#endif
+#if defined(TIM12)
+	case 12:
+		base = TIM12;
+		break;
```

Comment:
I would add these changes to a different commit

Reply:
yeah please separate them

Reply:
okay. i will separate.